.github/workflows/bazel.yml

```diff
@@ -451,16 +449,15 @@ jobs:
 
       - name: Output value of 'package_was_built'
         shell: bash
-        id: package_build_status
-        # This is necessary because the addnab/docker-run-action
-        # does not allow setting outputs from inside the container.
+        # This is necessary because the addnab/docker-run-action does not allow
+        # the setting of outputs or environment variables from inside the container.
         run: |
           PACKAGE_BUILD_STATUS="$(cat build_status.txt)"
           echo "$PACKAGE_BUILD_STATUS"
-          echo "$PACKAGE_BUILD_STATUS" >> $GITHUB_OUTPUT
+          echo "$PACKAGE_BUILD_STATUS" >> $GITHUB_ENV
 
       - name: Get magma version
-        if: ${{ needs.bazel_package.outputs.package_was_built == 'true' }}
+        if: ${{ env.package_was_built == 'true' }}
```

Comment:
Is this not the before state? I can't find any GitHub doc on step outputs that are not job outputs.

Reply:
What I imagine is to revert line 457, so that the step has an output again. I think that within one job, you can the refer to the step outputs with my suggestion

Reply:
Yes, that works. I think I have seen this before, but I cannot find it anywhere in the official docs.